packages/reactive-dict/reactive-dict-tests.js

```diff
@@ -0,0 +1,40 @@
+Tinytest.add('ReactiveDict - all() works', function (test) {
+  var all = {}, dict = new ReactiveDict;
+  Tracker.autorun(function() {
+    all = dict.all();
+  });
+  
+  test.equal(_.keys(all).length, 0);
+  
+  dict.set('foo', 'bar');
+  Tracker.flush();
+  test.equal(_.keys(all).length, 1);
```

Comment:
It seems like this would be better as just `test.equal(all, {foo: "bar"})` unless I'm incorrect that equal does a deep comparison?